backend/api/Database/Models/Mission.cs

```diff
@@ -58,7 +58,7 @@ public MissionStatus MissionStatus
 
         public DateTimeOffset EndTime { get; set; }
 
-        public TimeSpan EstimatedDuration { get; set; }
+        public int EstimatedDuration { get; set; }
```

Comment:
We dont want to change this to int i think, if you need the hours or minutes see:
https://learn.microsoft.com/en-us/dotnet/api/system.timespan?view=net-7.0